SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -1032,6 +1045,55 @@ public Infrastructure.CaseStatusType GetCaseStatusTypeWithFields(string type)
             return response.FirstOrDefault();
         }
 
+        public Infrastructure.CaseStatus CreateCaseStatus(CreateCaseStatusRequest request)
+        {
+
+            var statusType = _databaseContext.CaseStatusTypes
+                .Where(f => f.Name == request.Type)
+                .FirstOrDefault();
+
+            var caseStatus = new Infrastructure.CaseStatus()
+            {
+                PersonId = request.PersonId,
+                TypeId = statusType.Id,
+                StartDate = request.StartDate,
+                Notes = request.Notes,
+                CreatedBy = request.CreatedBy
+            };
+
+            _databaseContext.CaseStatuses.Add(caseStatus);
+            _databaseContext.SaveChanges();
```

Comment:
For more robust data changes, would be better to wrap all changes in a single transaction i.e. one SaveChanges() operation. This could be achieved by commenting out this line and making the changes higlighted below.